<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ed's duplicate files finder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0">
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="1,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item row="0" column="0" rowspan="3" colspan="2">
       <widget class="QTreeWidget" name="treeWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>300</number>
        </attribute>
        <column>
         <property name="text">
          <string>Path</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Corresponding file</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pb_addFolderContents">
        <property name="text">
         <string>Add folder...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Search in</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_folder"/>
      </item>
      <item>
       <widget class="QPushButton" name="pb_setFolder">
        <property name="text">
         <string>Select folder...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pbGo">
      <property name="text">
       <string>Go!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
